underscore/underscore.d.ts

```diff
@@ -179,6 +179,12 @@ interface UnderscoreStatic {
 		memo?: TResult,
 		context?: any): TResult;
 
+	reduce<T, TResult>(
+		list: _.Collection<T>,
```

Comment:
I think `object: _.Dictionary<T>` is rihgt, not `list: _.Collection<T>`.
Is this right?